tests/marker.py

```diff
@@ -28,3 +29,10 @@ def skip_wrapper(f):
         return pytest.mark.skipif(skip, reason=f'Requires Pytorch < {pt_version}')(f)
 
     return skip_wrapper
+
+def requires_jit_disabled():
```

Comment:
we make the reason an arg with default the string below, so that it can be edited depending on context. When we disable because of export, we say so, when it's because of mock, we say so